Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/ExploreFeedViewController.m

```diff
@@ -154,8 +151,7 @@ - (void)getNonprofitImagesFromBasket:(Basket*)basket onCell:(BasketTableViewCell
         [profFile getDataInBackgroundWithBlock:^(NSData * _Nullable data, NSError * _Nullable error) {
             UIImage *im;
             if (error != nil) {
-                NSLog(@"%@", error.localizedDescription);
-                im = [UIImage imageNamed:@"PlaceholderProfilePic"];
+]                im = [UIImage imageNamed:@"PlaceholderProfilePic"];
```

Comment:
Looks like there's an extra `]` at the beginning of this line, probably won't compile 😄